koji_containerbuild/cli.py

```diff
@@ -35,18 +35,32 @@
 DEFAULT_CHANNEL = 'container'
 
 
+def print_value(value, level, indent, suffix=''):
+    offset = ' ' * level * indent
+    print ''.join([offset, str(value), suffix])
+
+
+def print_result(result, level=0, indent=2):
+    if type(result) == list:
```

Comment:
yes. it's safer to do so, but I doubt koji will extend `list` and `dict` classes.